docs/uds-packages/README.md

```diff
@@ -0,0 +1,41 @@
+# Quick Start - UDS Packages
+
+This Quick Start is intended to provide the most concise and direct path to creating a Bronze UDS package.
+
+> [!TIP]
+> **Why?**
+> Defense Unicorns is focused on making software conveniently available to the warfighter. _wherever the mission may take them_.
+>
+> Creating a UDS package makes that piece of software compatible and available in the UDS ecosystem.
+
+<p align="center">
+    <img align="top" src=".images/boromir.png" alt="alt text" width="200">
+    <img src=".images/wat-no-not-that-wat.png" alt="alt text" width="200">
+</p>
+
+1. `git clone https://github.com/defenseunicorns/uds-package-template.git` then find and replace template variables
```

Comment:
Probably worth mentioning to `rm -rf .git/` and to `git init` to get rid of the template history as I didn't do that for my first couple of packages

Nit: Might also be worth mentioning `git clone https://github.com/defenseunicorns/uds-package-template.git uds-package-<package name>` so that it automatically goes into a directory to match the package repo name and keep organized